web/src/stream_data.js

```diff
@@ -740,29 +740,20 @@ export function create_sub_from_server_data(attrs) {
     delete attrs.subscribers;
 
     sub = {
-        name: attrs.name,
-        render_subscribers: !page_params.realm_is_zephyr_mirror_realm || attrs.invite_only === true,
-        subscribed: true,
+        render_subscribers: !page_params.realm_is_zephyr_mirror_realm || attrs.invite_only,
```

Comment:
Hmm, are you sure the `=== true` can be safely removed? I would worry it was there because of a concern that we wanted a value of `true|false` here, and not some other falsey value like `undefined` or `null` which `attrs.invite_only` could have had as a value.

Reply:
I checked all the possible code paths up until this line and it seems `invite_only` should always exist on `attrs` type.
But to be extra safe we can leave `=== true` in there but once we have more concrete `zod` schemas we can revisit this.